src/index.tsx

```diff
@@ -37,6 +37,7 @@ import { store } from '@/storage/store'
 // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
 // const root = createRoot(document.getElementById('root')!)
 
+// eslint-disable-next-line react/no-deprecated
```

Comment:
What is preventing us from using the non-deprecated way of rendering the dom?

Reply:
The scrollbar lib :)

<img width="579" alt="image" src="https://github.com/alephium/desktop-wallet/assets/1579899/e72cf390-4a19-4141-88a2-2eea2d604f62">

Reply:
arhhhhaarrrhrhhhrrr right.